tests/Api/Admin/ShippingMethodsTest.php

```diff
@@ -155,10 +155,14 @@ public function it_does_not_update_a_shipping_method_with_duplicate_locale_trans
             ], \JSON_THROW_ON_ERROR),
         );
 
-        $this->assertResponse(
+        $this->assertResponseViolations(
```

Comment:
Feels like I still should be able to specify response code, there at least two codes of 400 and 422 in use.